### ISSUE TITLE
Review how `create fuels` install dependencies

### ISSUE THREAD

**Issue by arboleya** [FuelLabs/fuels-ts#3481]
When I run `bun create fuels`, the generated app doesn't contain a `bun.lockb` file.

Only a after running:

```console
$ bun install
[0.40ms] ".env.local"
bun install v1.0.25 (a8ff7be6)
[10.35ms] migrated lockfile from package-lock.json

 127 packages installed [1.65s]
```

Same with `pnpm`, I only get the `pnpm-lock.yaml` file after running `pnpm install` inside the app.

It's worth also double-checking how `npm` is behaving.

**Comment by arboleya** [comment]
@danielbate @Dhaiwat10 Thoughts on this one?

I'm just trying to remember how this was initially done and if this behavior is expected.

**Comment by Dhaiwat10** [comment]
@arboleya there was an attempt to detect the package manager that the user used to initialize the project, but we did not succeed. I think as of now, the CLI always uses `npm` to install dependencies unless the user explicitly passes the package manager of their choice via a flag. (eg. `bun create fuels --bun`)

I noticed that `create-next-app` now auto-detects the package manager you use, so maybe there is a breakthrough there and we should re-visit this problem.
